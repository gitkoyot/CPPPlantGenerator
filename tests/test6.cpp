
#include "test6.h"

SingletonClass* SingletonClass::me=0;

/*
@startuml
class SingletonClass{
me   :   SingletonClass *
}

SingletonClass o-- SingletonClass
@enduml
*/
