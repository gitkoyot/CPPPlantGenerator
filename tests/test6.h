#ifndef TEST_6_H
#define TEST_6_H

class SingletonClass
{



    static SingletonClass *me;
};



#endif
