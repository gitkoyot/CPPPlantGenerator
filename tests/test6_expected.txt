@startuml
class SingletonClass{
{static}-me   :   SingletonClass *
}

SingletonClass o-- SingletonClass
@enduml

